client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -6,29 +6,43 @@ const useStyles = makeStyles((theme) => ({
     marginTop: theme.spacing(1),
   },
   label: {
-    fontSize: 19,
-    color: 'rgb(0,0,0,0.4)',
-    paddingLeft: '5px',
+    fontSize: 16,
+    color: 'rgb(0,0,0,0.7)',
+    paddingLeft: '1px',
+    fontFamily: 'Helvetica',
   },
   inputs: {
-    marginTop: '.8rem',
-    height: '2rem',
+    marginTop: '.3rem',
+    height: '2.5rem',
     padding: '5px',
+    paddingLeft: '15px',
```

Comment:
paddingLeft and padding right above here would just overwrite each other so instead maybe you can just write

`padding: 5px 5px 5px 15px`